docs/features/text-product-attribute.md

```diff
@@ -0,0 +1,21 @@
+# TextProductAttributeField
+
+Allows adding further details to a product. When used in combination with a [_PriceVariantsPart_](price-variants-part.md), it creates variants for the product. 
+
```

Comment:
I wrote this example as a clarification in another situation, but I figure we might as well include it here so people won't think that the field is useless without predefined values-
```suggestion
When not restricted to predefined values, it can be used to provide a text or note to the merchant. For example in a t-shirt store it may be used to supply custom text to be printed on the shirt.

```